srcpkgs/marvin/template

```diff
@@ -0,0 +1,56 @@
+# Template for Marvin
+pkgname=marvin
+version=18.16.0
+revision=1
+maintainer="Brenton Horne <brentonhorne77@gmail.com>"
+homepage="https://chemaxon.com/products/marvin"
+license="marvin"
+depends="oracle-jdk"
```

Comment:
Does this actually need `oracle-jdk` or would openjdk also works. If openjdk would also works, use `depends="virtual?java-enviroment"`

Reply:
I'd imagine either would, so I'll give 'er an edit, thanks for your attention to detail :).

Reply:
Are you sure about this? As the latest Travis test results do not like this depends line &mdash; https://travis-ci.org/void-linux/void-packages/jobs/411473503.

Reply:
You should not copy my typing mistakes :laughing: . I missed a `n` `virtual?java-environment`

Reply:
Rofl, what can I say, it's my basic assumption you's are right and I am wrong.